README.md

```diff
@@ -16,21 +18,50 @@ optimize &nbsp; c₀ + cᵀ x + ½ xᵀ Q x
 subject to &nbsp; L ≤ Ax ≤ U and ℓ ≤ x ≤ u,
 </p>
 
-where "optimize" means either "minimize" or "maximize", c₀ ∈ ℝ is a constant term, c ∈ ℝⁿ is the linear term, Q = Qᵀ is the *n×n* positive semi-definite quadratic term, L is the vector of lower constraint bounds, A is the constraint matrix, U is the vector of upper constraint bounds, ℓ is the vector of lower bounds, and u is the vector of upper bounds.
+where:
+* "optimize" means either "minimize" or "maximize"
+* `c₀` ∈ ℝ is a constant term, `c` ∈ ℝⁿ is the linear term, `Q = Qᵀ` is the *n×n* positive semi-definite quadratic term,
```

Comment:
Didn't we decide that Q could in fact be anything symmetric?

Reply:
Yes, I missed that. Thanks